front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/index.tsx

```diff
@@ -35,8 +36,8 @@ const TextQuestion = ({
         />
       </Box>
       <Box flex="1">
-        {!isAnalysisEnabled && showAnalysis && <AnalysisUpsell />}
-        {isAnalysisEnabled && showAnalysis && !hasOtherResponses && (
+        {!isAnalysisAllowed && <AnalysisUpsell />}
+        {isAnalysisEnabled && !hasOtherResponses && (
```

Comment:
Just a detail, but to not confuse the double meaning of the word `enabled`, I typically use the word `active` to denote that it's both allowed and enabled. So I think `isAnalysisActive` is clearer.

Reply:
Fair but I think we use enabled in variable names throughout the FE. I'll add it to my CDM list